### PR TITLE
Chore: Federation E2E tests for messaging + admin

### PR DESCRIPTION
**Pull request by MarcosSpessatto**
## Proposed changes (including videos or screenshots)

Depends on https://github.com/RocketChat/Rocket.Chat/pull/27115

Jira: FED-41

### CODE REVIEW

**Comment by AlenDavid** [comment]
Same as here, we have a check failing `
Build and Test / test-ee (14.19.3, 4.4) (pull_request)`

File: apps/meteor/tests/e2e/federation/page-objects/fragments/home-content.ts [hunk 1]
```diff
@@ -25,16 +23,72 @@ export class FederationHomeContent {
...
+		await this.page.locator('.vrec-dialog button.record').click();
+		await this.page.waitForTimeout(5000);
+		await this.page.locator('.vrec-dialog button.record').click();
+		await this.page.locator('.vrec-dialog button.ok').click();
+		await this.btnModalConfirm.click();
+	}
+
 	async dispatchSlashCommand(text: string): Promise<void> {
 		await this.page.locator('[name="msg"]').type(text);
+		await this.page.waitForTimeout(2000);
```
Comment (jessicaschelly) [inline apps/meteor/tests/e2e/federation/page-objects/fragments/home-content.ts:87]: Is it possible to change the _waitForTimeout_ for something more specific? So it's more clear what we are waiting  
e.g: waitForSelector, waitForEvent

```suggestion
		await this.page.locator('//*[contains(@class, "rc-message-box__icon") and contains(@class, "rc-message-box__send") and contains(@class, "js-send")]').waitFor()
```

**Comment by pierre-lehnen-rc** [comment]
The test image you included is owned by [DreamsTime](https://www.dreamstime.com/) and we don't have the license to use it.

File: apps/meteor/tests/e2e/federation/page-objects/fragments/home-content.ts (hunk 1 above)
Comment (MarcosSpessatto) [reply apps/meteor/tests/e2e/federation/page-objects/fragments/home-content.ts]: Thanks for the tip, I've removed it wherever and whenever it was possible, I just kept it in some specific cases.

**Review by jessicaschelly — APPROVED** [review]

**Pull request merged**